kubernetes/e2e_test/test_client.py

```diff
@@ -154,6 +169,32 @@ def test_pod_apis(self):
         resp.update(timeout=5)
         self.assertFalse(resp.is_open())
 
+        resp = stream(api.connect_post_namespaced_pod_exec, name, 'default',
+                      command='/bin/sh',
+                      stderr=True, stdin=True,
+                      stdout=True, tty=False,
+                      binary=True,
+                      _preload_content=False)
```

Comment:
add a test case that is the same as this one, but with _preload_content=True ?

Reply:
The test above this one leaves _preload_test as its default value of True. Using update() and write_stdin(), read_stdout(), and read_stderr() are not valid when this is the case.

Reply:
ok